heron/common/src/python/utils/config.py

```diff
@@ -133,9 +128,15 @@ def get_classpath(jars):
 def get_heron_dir():
   """
   This will extract heron directory from .pex file.
+
+  For example,
+  when __file__ is '/Users/heron-user/bin/heron/heron/common/src/python/utils/config.pyc', and
+  its real path is '/Users/heron-user/.heron/bin/heron/common/src/python/utils/config.pyc',
+  the internal variable ``path`` would be '/Users/heron-user/.heron', which is the heron directory
+
   :return: root location for heron-cli.
   """
-  path = "/".join(os.path.realpath(__file__).split('/')[:-7])
+  path = "/".join(os.path.realpath(__file__).split('/')[:-8])
   return normalized_class_path(path)
 
```

Comment:
Thanks for the documentation. Not related to your PR, but I want to say this function has always been quite flaky. Because of the way this function uses to get ``path``, `heron-ui` cannot have `version` subcommand (#1175). In this PR, @taishi8117 moved this file into a new subfolder (which is OK) and he had to increment these hardcoded levels.